packages/formation/sass/core.scss

```diff
@@ -27,6 +27,11 @@
 @import "~uswds/src/stylesheets/core/fonts";
 @import "~uswds/src/stylesheets/core/grid";
 @import "~uswds/src/stylesheets/core/utilities";
+
+// VA mixins. Here so that we can override USWDS mixins.
+// Specifically, the "focus" mixin.
+@import "base/b-mixins";
+
```

Comment:
This allows us to override the focus mixin. This seems like a pattern we should do more often, such as by overriding their [heading mixins](https://github.com/uswds/uswds/blob/c98f69c879d0fc9e2e2cc72ee9ee9743c1971d39/src/stylesheets/core/_utilities.scss#L7) with [ours ](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/base/_b-mixins.scss#L27).